bin/jeprof.in

```diff
@@ -4698,7 +4698,7 @@ sub ParseLibraries {
       $offset = HexExtend($3);
       $lib = $4;
       $lib =~ s|\\|/|g;     # turn windows-style paths into unix-style paths
-    } elsif ($l =~ /^\s*($h)-($h):\s*(\S+\.so(\.\d+)*)/) {
+    } elsif ($l =~ /^\s*($h)-($h):\s*(.*$)/) {
```

Comment:
Match for library mappings on macOS dump. The library names can have .dylib extension or no extension at all.